MalwareAnalyzer/views/domain_check.py

```diff
@@ -105,7 +105,12 @@ def get_domains(urls):
         domains = []
         for url in urls:
             parsed_uri = urlparse(url)
+            if not urlparse(url).scheme:
+                url = "//"+url
+                parsed_uri = urlparse(url)
             domain = '{uri.netloc}'.format(uri=parsed_uri)
+            if domain.startswith('www.'):
```

Comment:
This is a problem. We have malware signature with `www.` and that will break it.

Reply:
Can work if you change statswith to contains in here: https://github.com/MobSF/Mobile-Security-Framework-MobSF/blob/master/MalwareAnalyzer/views/domain_check.py#L86

Reply:
corrected :)